<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pdokbaggeocoder_form</class>
 <widget class="QDialog" name="pdokbaggeocoder_form">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>378</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>PDOK BAG Geocoder</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="13" column="0" colspan="3">
    <widget class="QLineEdit" name="shapefilename">
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="housenumber_label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Huisnummer</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2" colspan="2">
    <widget class="QComboBox" name="housenumber"/>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="browse_infile">
     <property name="text">
      <string>Select...</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="city_label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Woonplaats</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2" colspan="2">
    <widget class="QComboBox" name="city"/>
   </item>
   <item row="10" column="1">
    <widget class="QRadioButton" name="radio_list">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>een woonplaats</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="2">
    <widget class="QComboBox" name="address"/>
   </item>
   <item row="10" column="0">
    <widget class="QRadioButton" name="radio_column">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>selecteer een kolom of</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QLineEdit" name="infilename">
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="address_sublabel">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>huisnummertoevoegingen worden genegeerd</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="address_label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Adres (met huisnummer) of Postcode</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" rowspan="2" colspan="3">
    <widget class="QLabel" name="label_add_column_selection">
     <property name="text">
      <string>Koppel de kolomvelden uit het CSV bestand: </string>
     </property>
    </widget>
   </item>
   <item row="13" column="3">
    <widget class="QPushButton" name="browse_shapefile">
     <property name="text">
      <string>Save as...</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="housenumber_sublabel">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>optioneel, als huisnummers apart staan</string>
     </property>
    </widget>
   </item>
   <item row="14" column="2" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_select_file">
     <property name="text">
      <string>Selecteer CSV bestand:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="4">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLineEdit" name="notfoundfilename"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>pdokbaggeocoder_form</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>pdokbaggeocoder_form</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
